docs/sections/gha_workflows.md

```diff
@@ -187,6 +187,14 @@ refer to an action on a public Git repository or Docker container registry:
 
 -->
 
+### Using Private repositories of `Github` in an action
+
+When creating an action block we can also use an action defined in Github private repository. To use private repository you need to create Personal access token from <a href="https://github.com/settings/tokens"> here </a> with permissions to access private repositories and set it as an environment variable using
```

Comment:
couple of changes (note that readthedocs accepts markdown):

We can make use of actions located in private Github repositories by defining a `GITHUB_API_TOKEN` environment variable that the `popper run` command can inspect. To accomplish this, the repository referenced in the `uses` attribute is assumed to be private and, to access it, an API token from Github is needed (see instructions [here](https://github.com/settings/tokens)). The token needs to have permission to read the private repository in question. To define the `GITHUB_API_TOKEN` variable:

```
export GITHUB_API_TOKEN=access_token_here
popper run
```

If the access token doesn't have permissions to access private repositories, the `popper run` command will fail.